modules/components/pages/outputs/quest_db.adoc

```diff
@@ -0,0 +1,423 @@
+= questdb
+:type: output
+:status: experimental
+:categories: ["Services"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Pushes messages to a QuestDB table.
+
+[tabs]
+======
+Common::
++
+--
+```yml
+# Common configuration fields, showing default values
+output:
+  label: ""
+  questdb:
+    max_in_flight: 64
+    batching:
+      count: 0
+      byte_size: 0
+      period: ""
+      check: ""
+    address: localhost:9000 # No default (required)
+    username: "" # No default (optional)
+    password: "" # No default (optional)
+    token: "" # No default (optional)
+    table: trades # No default (required)
+    designated_timestamp_field: "" # No default (optional)
+    designated_timestamp_unit: auto
+    timestamp_string_fields: [] # No default (optional)
+    timestamp_string_format: Jan _2 15:04:05.000000Z0700
+    symbols: [] # No default (optional)
+    doubles: [] # No default (optional)
+    error_on_empty_messages: false
+```
+--
+Advanced::
++
+--
+```yml
+# All configuration fields, showing default values
+output:
+  label: ""
+  questdb:
+    max_in_flight: 64
+    batching:
+      count: 0
+      byte_size: 0
+      period: ""
+      check: ""
+      processors: [] # No default (optional)
+    tls:
+      enabled: false
+      skip_cert_verify: false
+      enable_renegotiation: false
+      root_cas: ""
+      root_cas_file: ""
+      client_certs: []
+    address: localhost:9000 # No default (required)
+    username: "" # No default (optional)
+    password: "" # No default (optional)
+    token: "" # No default (optional)
+    retry_timeout: "" # No default (optional)
+    request_timeout: "" # No default (optional)
+    request_min_throughput: 0 # No default (optional)
+    table: trades # No default (required)
+    designated_timestamp_field: "" # No default (optional)
+    designated_timestamp_unit: auto
+    timestamp_string_fields: [] # No default (optional)
+    timestamp_string_format: Jan _2 15:04:05.000000Z0700
+    symbols: [] # No default (optional)
+    doubles: [] # No default (optional)
+    error_on_empty_messages: false
+```
+--
+======
+
+IMPORTANT: Redpanda recommends that you enable the dedupe feature on the QuestDB server. For more information about about deploying, configuring, and using QuestDB, see the https://questdb.io/docs/[QuestDB documentation]. 
+
+== Performance
+
+For improved performance, this output sends multiple messages in parallel. You can tune the maximum number of in-flight messages or message batches, using the `max_in_flight` field.
+
+This output can also send message batches. You can configure batches at both the input and output level. For more information, see xref:configuration:batching.adoc[Message Batching].
```

Comment:
The relationship between 'in-flight messages' and message batches is unclear here.

> You can tune the maximum number of in-flight messages or message batches, using the max_in_flight field.

This implies that message batches is a synonym of in-flight messages.

> This output can also send message batches. You can configure batches at both the input and output level. For more information, see xref:configuration:batching.adoc[Message Batching].

Then, this implies that message batches are entirely different. We also don't mention the batching.* configuration options here but I think it would be useful.

Reply:
Updated to make this clearer, as my rewording was inaccurate.

I think adding configuration options for batching is part of a site wide change, see https://redpandadata.atlassian.net/browse/DOC-629.